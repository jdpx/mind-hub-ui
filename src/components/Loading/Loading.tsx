import React from 'react'

interface Props {
    testid?: string
}

export default function Loading({ testid = 'loading' }: Props) {
    return <div data-testid={testid}>Loading</div>
}
Loading.displayName = 'Loading'
