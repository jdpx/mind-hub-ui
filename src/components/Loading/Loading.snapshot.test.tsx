import React from 'react'
import { render } from '@testing-library/react'

import Loading from './Loading'

describe('Loading Snapshots', () => {
    it('renders loading panel', () => {
        const { asFragment } = render(<Loading />)
        expect(asFragment()).toMatchSnapshot()
    })
})
