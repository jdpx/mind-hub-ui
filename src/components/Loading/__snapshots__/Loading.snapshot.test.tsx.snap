// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Snapshots renders loading panel 1`] = `
<DocumentFragment>
  <div
    data-testid="loading"
  >
    Loading
  </div>
</DocumentFragment>
`;
